docs/src/single_column_prospect.md

```diff
@@ -0,0 +1,61 @@
+# Single Column Models
+`ClimaAtmos.jl` supports several canonical test cases that are run in a single column model designed to verify how PROSPECT (EDMF) performs against other test cases. These cases include variants of `bomex`, `dycoms`, `rico`, `soares`, and `trmm` and can be found in the `config/model_configs` directory. To run, for example, the BOMEX test case execute the following:
+```bash
+julia --project=examples examples/hybrid/driver.jl --config_file config/model_configs/prognostic_edmfx_bomex_column.yml --job_id bomex
+```
+It may also be helpful to run in interactive mode to be able to examine the simulation object,  debug, and develop the code further. To enter debug mode run `julia --project=examples` and then in the REPL run: 
```

Comment:
```suggestion
It may also be helpful to run in interactive mode to be able to examine the simulation object, debug, and develop the code further. To enter debug mode run `julia --project=examples` and then in the REPL run: 
```